aten/src/ATen/native/xnnpack/Common.h

```diff
@@ -31,7 +78,7 @@ struct Layout final {
       }
 
       // Handle the case where batch size is zero.
-      int64_t batch = std::max<int64_t>(1, tensor[0]);
+      int64_t batch = tensor[0];
```

Comment:
Seems like this might be a bug in the old version.  But also, unrelated to this diff, is this function taking an argument called "tensor" that is meant to be a tensor shape?  That's confusing.

Reply:
Yes, this was a bug in the older version. This is to handle batch size 0.